mne/coreg.py

```diff
@@ -561,6 +568,14 @@ def _find_mri_paths(subject, skip_fiducials, subjects_dir):
                 dup.append(pformat(surf_fname, name=name))
     del surf_dup_name, name, path, dup, hemi
 
+    # transform files (talairach)
+    paths['transforms'] = []
+    transform_fname = os.path.join(mri_transforms_dirname, 'talairach.xfm')
+    path = transform_fname.format(subjects_dir=subjects_dir, subject=subject)
+    if os.path.exists(path):
+        paths['transforms'].append(transform_fname)
+    del transform_fname, path
```

Comment:
Is this `del` statement left over from debugging?

Reply:
No, it's more for code safety to ensure if `path` is used later it's using the correct one